gamestonk_terminal/stocks/discovery/yahoofinance_view.py

```diff
@@ -77,3 +77,38 @@ def display_losers(num_stocks: int, export: str):
         "losers",
         df_losers,
     )
+
+
+def display_undervalued(num_stocks: int, export: str):
+    """Display most undervalued growth stock. [Source: Yahoo Finance]
+    Parameters
+    ----------
+    num_stocks: int
+        Number of stocks to display
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+    df = yahoofinance_model.get_undervalued().head(num_stocks)
+    df.dropna(how="all", axis=1, inplace=True)
+    df = df.replace(float("NaN"), "")
+
+    if df.empty:
+        print("No data found.")
+    else:
+        print(
+            tabulate(
+                df,
```

Comment:
Can we have the .head() at the tabulate time?

This allows so that the dataframe remains complete if the user wants to export this information